content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```diff
@@ -123,6 +125,33 @@ your cluster. Those fields are:
   See [Label Selectors](/docs/concepts/overview/working-with-objects/labels/#label-selectors)
   for more details.
```

Comment:
We have improved our API docs a lot to include definitions of "topology domain", "eligible domain", and others. For example: https://github.com/kubernetes/api/blob/c8f06018bfc8e3c8213c53ec87b7fa4e2b9121ab/core/v1/types.go#L3396

Could you add those in this doc?

Maybe we could add the definitions before talking about the field themselves.

Reply:
It's also OK to link to https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#scheduling (ie, in Markdown it'd be `[anchor text](/docs/reference/kubernetes-api/workload-resources/pod-v1/#scheduling)`) if that's useful. No need to repeat the API reference word-for-word.